src/main/java/org/broadinstitute/consent/http/resources/ConsentManageResource.java

```diff
@@ -25,16 +25,6 @@ public ConsentManageResource(ConsentService consentService) {
         this.consentService = consentService;
     }
 
-    @Deprecated
-    @GET
-    @Path("/manage")
-    @RolesAllowed(ADMIN)
-    @Produces(MediaType.APPLICATION_JSON)
-    public Response getConsentManage(@Auth AuthUser authUser) {
-        List<ConsentManage> consentManages = consentService.describeConsentManage(authUser);
-        return Response.ok().entity(consentManages).build();
-    }
-
```

Comment:
I removed this endpoint as it was a difficult code path to maintain and it was already deprecated. Let me know if this should not be removed yet.

Reply:
This also makes ConsentManage unused. Should I go ahead and remove the model/mapper/etc.?

Reply:
Yes, this can be removed along with anything downstream that is unused. We've had several releases since the new admin navigation screen release, so these are now completely outdated.

Reply:
great, will do